src/gsy_e/models/matching_engine_matcher/__init__.py

```diff
@@ -0,0 +1,12 @@
+from gsy_e.models.matching_engine_matcher.matching_engine_internal_matcher import \
+    MatchingEngineInternalMatcher
+from gsy_e.models.matching_engine_matcher.matching_engine_external_matcher import \
+    MatchingEngineExternalMatcher
+from gsy_e.models.matching_engine_matcher.matching_engine_matcher_interface import \
+    MatchingEngineMatcherInterface
+
+__all__ = [
+    "MatchingEngineMatcherInterface",
+    "MatchingEngineExternalMatcher",
+    "MatchingEngineInternalMatcher"
```

Comment:
Please also add the `MatchingEngineInternalForwardMarketMatcher`. 
Thanks a lot!